pkgs/by-name/vi/victorialogs/package.nix

```diff
@@ -34,11 +34,9 @@ buildGoModule (finalAttrs: {
 
   postPatch = ''
     # Allow older go versions
-    substituteInPlace go.mod \
-      --replace-fail "go 1.25.2" "go ${finalAttrs.passthru.go.version}"
+    sed -i go.mod -e 's/^go .*/go ${finalAttrs.passthru.go.version}/'
```

Comment:
Please only strip the patch version. Using an older go minor version might cause some hard to debug errors.

Reply:
As I was just copying what was done in the victoriametrics package (like statet in the PR description) to keep both in logic kinda the same and I am not very into sed magic I have doubts being able to produce the request (tho it sounds reasonable in overall to me) I will just drop that commit for now (later when I am at my desktop).